### PR TITLE
F821 - undefined name 'name'

### LINKED ISSUE #1274: Remove the queso module?
While trying to write tests for the queso module, I could not easily find the source code online. It seems that the project is now abandoned. Shall we remove this module?

### PR DESCRIPTION
**Pull request by guedou**
This PR fixes real bugs found using flake8. #1277 serie.

I decided to remove the queso module as it is almost impossible to download, install and use queso nowadays. This will close #1274.

FTR, here is the patch that fixes the F821 issue with the queso module:
```
diff --git a/scapy/modules/queso.py b/scapy/modules/queso.py
index d5b94a8..835d694 100644
--- a/scapy/modules/queso.py
+++ b/scapy/modules/queso.py
@@ -47,9 +47,9 @@ class QuesoKnowledgeBase(KnowledgeBase):
                 if not l or l[0] == ';':
                     continue
                 if l[0] == '*':
+                    name = l[1:].strip()
                     if p is not None:
                         p[""] = name
-                    name = l[1:].strip()
                     p = self.base
                     continue
                 if l[0] not in list("0123456"):

diff --git a/test/regression.uts b/test/regression.uts
index bcfbc41..5c526f2 100644
--- a/test/regression.uts
+++ b/test/regression.uts
@@ -10193,3 +10193,30 @@ assert "RTPExtension" not in parsed
 #    assert(output == expected)
 #
 #test_snmpwalk("secdev.org")
+
+= Test Scapy Queso module
+
+load_module("queso")
+import tempfile
+filename = tempfile.mktemp()
+fdesc = open(filename, "w")
+queso_conf = """
+; From queso.conf(5)
+
+* Linux 2.1.xx/2.2.xx
+0 1 1 7FB8 SA
+1 0 0 0 R
+2 - - - -
+3 0 0 0 R
+4 1 1 7FB8 SA
+5 - - - -
+6 1 1 7FB8 SA
+"""
+fdesc.write(queso_conf)
+fdesc.close()
+queso_kdb.reload(filename)
+os.unlink(filename)
+res = queso_search(["1 1 7FB8 12", "0 0 0 4", "- - - -",
+                    "0 0 0 4", "1 1 7FB8 12", "- - - -",
+                    "1 1 7FB8 12"])
+assert res == ["Linux 2.1.xx/2.2.xx"]
```

### CODE REVIEW

File: scapy/arch/common.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def _legacy_bpf_pointer(tcpdump_lines):
     bpf = b""
     for l in tcpdump_lines[1:]:
         if six.PY2:
-            int_type = long
+            int_type = long  # noqa: F821
```
Comment (guedou) [inline scapy/arch/common.py]: flake8 won't complain if launched with a Python3 interpreter.

File: tox.ini [hunk 2]
```diff
@@ -43,6 +43,9 @@ commands = flake8 scapy/
 
 # flake8 configuration
 [flake8]
-select = E23,E30,E20,E12,W29,E25,E26
+disabel-noqa = True
```
Comment (gpotter2) [inline tox.ini]: Seems wrong:

> If we ever want to disable Flake8 respecting # noqa comments, we can can refer to flake8 --disable-noqa.

Shoudln't it be disab**le**-noqa (not disab**el**-noqa)

Comment (p-l-) [reply tox.ini]: Agreed, you should remove this line so that your noqa comments work.

**Comment by guedou** [comment]
Fixed and rebased.

**Comment by gpotter2** [comment]
This should be merged before the other PepIn PRs, so that we get rid of queso before applying others (conflicts and stuff)

**Comment by guedou** [comment]
Do you approve it?

**Pull request merged**